app/simformat.hs

```diff
@@ -102,6 +108,7 @@ main = do
     PrintNumericVersion _ _       -> putStrLn $ showVersion Paths.version
     InPlace file regroup validate -> format optsVerbose config (Just file) regroup validate
     Repo regroup validate         -> format optsVerbose config Nothing regroup validate
+    Editor regroup _                    -> formatStdIn regroup
```

Comment:
Line up the `->`?

Reply:
Update the validate help to say that it is ignored by editor mode? Either that or add to the help for this option that validate is ignored